stream.go

```diff
@@ -737,10 +742,22 @@ func bulkAppendFields(w io.Writer, ws *xlsxWorksheet, from, to int) {
 // is written to the temp file with Sync, which may return an error.
 // Therefore, Sync should be periodically called and the error checked.
 type bufferedWriter struct {
+	tmpDir string
+
 	tmp *os.File
 	buf bytes.Buffer
 }
 
+// getTempDir returns the temporary directory for creating temporary files, if
+// the value is empty, the system default temporary directory which is calculated
+// by os.TempDir() will be used.
+func (bw *bufferedWriter) getTempDir() string {
```

Comment:
This function also can be removed. Hold on, I will make some updates based on your branch.